packages/components/pages/f-contact-preferences/src/constants.js

```diff
@@ -0,0 +1,11 @@
+export const UPDATE_PREFERENCES = 'updatePreferences';
```

Comment:
I have added all the constants here that I am using for subsequent prs that will follow.